src/evo/deterministicmns.cpp

```diff
@@ -930,7 +930,7 @@ bool CDeterministicMNManager::BuildNewListFromBlock(const CBlock &block, const C
     }
 
     mnListRet = std::move(newList);
-    UpdateLLMQParams(mnListRet.GetAllMNsCount(), nHeight, sporkManager.IsSporkActive(SPORK_21_LOW_LLMQ_PARAMS));
+    UpdateLLMQParams(mnListRet.GetAllMNsCount(), pindexPrev, sporkManager.IsSporkActive(SPORK_21_LOW_LLMQ_PARAMS));
```

Comment:
This previously passed in height, which was pindexPrev->nHeight + 1.  See any issues with using nHeight?

Reply:
I have updated this logic to use height + 1.